lib/sql.js

```diff
@@ -165,13 +168,13 @@ function tablesToSQL(tables) {
 
     if (baseTable.db && baseTable.db !== null) str = baseTable.db + '.' + str;
     if (baseTable.as !== null) str += ' AS ' + identifierToSql(baseTable.as);
-
+    
```

Comment:
Don't need this extra whitespace I don't think.

Reply:
Updated